model-api/src/commonMain/kotlin/org/modelix/model/api/ContextValue.kt

```diff
@@ -13,6 +13,7 @@
  */
 package org.modelix.model.api
 
+@Deprecated("use org.modelix.kotlin.utils.ContextValue from org.modelix:kotlin-utils")
 expect class ContextValue<E> {
```

Comment:
Looks like this class is a subset of the class from the new utils package. Could this deprecation be made easier for clients if the new implementation was type-aliased here? That would also avoid having two implementations in parallel?

Or can we treat the whole context value as being something internal and just avoid the deprecation cycle? I can't remember accessing that from client code.

Reply:
The class is used by Java code written in MPS. Making it a type-alias would be a breaking change.